error-handling-demos/config/databases.yaml

```diff
@@ -0,0 +1,10 @@
+Databases:
+  stock:
+    Url: jdbc:hsqldb:mem:StockDb
+    Driver: org.hsqldb.jdbcDriver
+  invalidDB:
+    Url: jdbc:postgresql://null/InvalidDB
+    Driver: org.postgresql.Driver
+    UserName: ''
+    Properties:
+      This DB is invalid on purpose: so it can be used to throw an error
```

Comment:
hm.. does this work?

Reply:
haha good eye 🙈
I will check that 😉

Reply:
![grafik](https://user-images.githubusercontent.com/42733123/115404473-a86d7980-a1ed-11eb-81b6-40922b4aa28d.png)
![grafik](https://user-images.githubusercontent.com/42733123/115404518-b15e4b00-a1ed-11eb-8a0e-c19471a404e9.png)
![grafik](https://user-images.githubusercontent.com/42733123/115404545-b7ecc280-a1ed-11eb-9dc3-fee166e2d152.png)

Reply:
Looks good, I think

Reply:
Same what happens on 9.2 engine 👍